src/plugins/data/public/ui/query_string_input/query_string_input.tsx

```diff
@@ -191,8 +189,7 @@ export default class QueryStringInputUI extends Component<Props, State> {
           selectionEnd,
           signal: this.abortController.signal,
         })) || [];
-
-      return [...suggestions, ...recentSearchSuggestions];
+      return [...suggestions];
```

Comment:
i thought autocomplete only applies to the query editor, does it need to change query string input as well?

Reply:
testing changes, also already been included in cleanup